ghastcoiler/minions/rank_2.py

```diff
@@ -0,0 +1,201 @@
+                         **kwargs)
+
+
+class OldMurkEye(Minion):
+    def __init__(self, **kwargs):
+        super().__init__(name="OldMurkEye",
+                         rank=2,
+                         base_attack=2,
+                         base_defense=4,
+                         types=[MinionType.Murloc],
+                         **kwargs)
+
+    def additional_attack(self, own_board, opposing_board):
+        # TODO: +1 attack for each other murloc on the battlefield
+        pass
+
+
+class PogoHopper(Minion):
+    def __init__(self, **kwargs):
+        super().__init__(name="Pogo-Hopper",
+                         rank=2,
+                         base_attack=1,
+                         base_defense=1,
+                         types=[MinionType.Mech],
+                         **kwargs)
+
+        # TODO add battlecry
+
+
+class RatPack(Minion):
+    def __init__(self, **kwargs):
+        super().__init__(name="RatPack",
+                         rank=2,
+                         base_attack=2,
+                         base_defense=2,
+                         types=[MinionType.Beast],
+                         base_deathrattle=RatPackDeathrattle(),
+                         **kwargs)
+
+
+class ScavengingHyena(Minion):
+    def __init__(self, **kwargs):
+        super().__init__(name="Scavenging Hyena",
+                         rank=2,
+                         base_attack=2,
+                         base_defense=2,
+                         types=[MinionType.Beast],
+                         **kwargs)
+
+
+class SpawnofNZoth(Minion):
+    def __init__(self, **kwargs):
+        super().__init__(name="Spawn of N'Zoth",
+                         rank=2,
+                         base_attack=2,
+                         base_defense=2,
+                         base_deathrattle=SpawnofNZothDeathrattle(),
+                         **kwargs)
+
+
+class StewardofTime(Minion):
+    def __init__(self, **kwargs):
+        super().__init__(name="Steward of Time",
+                         rank=2,
+                         base_attack=3,
+                         base_defense=4,
+                         types=[MinionType.Dragon],
+                         **kwargs)
+
+    # TODO: Add Selling mechanic
+
+
+class UnstableGhoul(Minion):
+    def __init__(self, **kwargs):
+        super().__init__(name="Unstable Ghoul",
+                         rank=2,
+                         base_attack=1,
+                         base_defense=3,
+                         base_taunt=True,
+                         base_deathrattle=UnstableGhoulDeathrattle(),
+                         **kwargs)
+
+class WaxriderTogwaggle(Minion):
+    def __init__(self, **kwargs):
+        super().__init__(name="Waxrider Togwaggle",
+                         rank=2,
+                         base_attack=1,
+                         base_defense=2,
+                         **kwargs)
+
+    # TODO: Add the Increment attribute
```

Comment:
Can be done using the on_other_death method which gets passed the minion that died so we can check if it was a Dragon

Reply:
"Whenever a friendly Dragon kills an enemy, gain +2/+2."

It feels like the variable "other_minion" should be our own minion that just killed an enemy and we check if our minion was a Dragon.